lib/entities/parse_address_from_domain.dart

```diff
@@ -104,6 +107,15 @@ class AddressResolver {
           return ParsedAddress.fetchOpenAliasAddress(record: record, name: text);
         }
       }
+      if (text.contains(".")) {
+        var wallet = getIt.get<AppStore>().wallet!;
+        if (wallet.type == WalletType.ethereum) {
+          final address = await ethereum!.fetchEnsAddress(wallet, text);
+          if (address.isNotEmpty) {
+            return ParsedAddress.fetchEnsAddress(name: text, address: address);
+          }
+        }
+      }
```

Comment:
```
      if (text.endsWith(".eth")) {
        final address = await fetchEnsAddress(text, ticker);
        return ParsedAddress.fetchEnsAddress(name: text, address: address);
      }
```

If these services are resolved in order (top to bottom), then move ENS above OpenAlias, so we can skip an attempted ENS lookup if the domain is `.ens`. We don't need to do an OpenAlias lookup if the TLD is an Unstoppable/ENS TLD, since these TLDs don't exist on the "Regular" internet.